### PR TITLE
add surfingkeys viebrc to help

### PR DESCRIPTION
**Pull request by ottopiramuthu**

### CODE REVIEW

File: app/examples/surfingkeys [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+set permissiondisplaycapture=allow
+set permissionmediadevices=allow
+set permissionmicrophone=ask
+set permissionpersistentstorage=allow
+set redirecttohttp
+set splitbelow
+set tabclosefocusright
+set tabreopenposition=previous
+set windowtitle=title
+set showcmd
```
Comment (Jelmerro) [inline app/examples/surfingkeys:19]: `showcmd` is enabled by default in Vieb so we can leave this out, aside from that your PR is great!

Comment (ottopiramuthu) [reply app/examples/surfingkeys]: I've removed it in the latest commit.

**Comment by Jelmerro** [comment]
Thanks!

**Comment by ottopiramuthu** [comment]
I will work on a [Pentadactyl](https://en.wikipedia.org/wiki/Pentadactyl) (in Pale Moon) viebrc next. I can already tell from the help page that this will take a bit longer to complete than the others, so I probably won't be working on anything else for a while.

**Pull request merged**